src/commands/transaction/list.ts

```diff
@@ -27,10 +27,34 @@ interface PromptAnswers {
   propertyId?: string;
 }
 
+const displayConciseTransactions = (response: SuccessResponse) => {
+  const transactions = response.data.payload as ListResponsePayload;
+  if (!transactions.length) {
+    return console.log(chalk.cyan("No transactions to display"));
+  }
+
+  const table = display.table.createCompact({
+    head: ["Trans ID", "Title", "Amount", "Property"],
+    colWidths: [28, 18, 10, 12],
+    colAligns: ["center", "left", "left", "left"],
+  });
+  transactions.forEach(
+    ({ id, title, amountChange, property }) => {
+      table.push([
+        id,
+        title,
+        display.values.formatPointsChange(amountChange),
+        property.name,
+      ]);
+    }
+  );
+  display.table.print(table);
+};
+
 const displayTransactions = (response: SuccessResponse) => {
   const transactions = response.data.payload as ListResponsePayload;
   if (!transactions.length) {
-    return console.log("Listed 0 transactions.");
+    return console.log(chalk.cyan("No transactions to display"));
   }
```

Comment:
I don't see a reason for this change, I think the original message might be good enough. Just want to make sure that I'm not missing context, what triggered you to make this copy adjustment?

Reply:
Oh that is just from some experiences with other applications using such language. Not really a big deal just feels more comfortable to read. Could be changed back any time. :)

Reply:
It'll be nice if we can change this back, to stay consistent with many tools that the developer uses.